indexers/social_feed/social_feed_reposts_v9.js

```diff
@@ -0,0 +1,450 @@
+      if (posts.length == 0) {
+        return;
+      }
+      const post = posts[0];
+      let accountsLiked =
+          post.accounts_liked.length === 0
+              ? post.accounts_liked
+              : JSON.parse(post.accounts_liked);
+
+      console.log(accountsLiked);
+
+      let indexOfLikeAuthorAccountIdInPost =
+          accountsLiked.indexOf(likeAuthorAccountId);
+      if (indexOfLikeAuthorAccountIdInPost > -1) {
+        accountsLiked.splice(indexOfLikeAuthorAccountIdInPost, 1);
+        // Call GraphQL mutation to update a post's liked accounts list
+        await context.db.Posts.update(
+            { id: postId },
+            { accounts_liked: JSON.stringify(accountsLiked) }
+        );
+      }
+      // Call GraphQL mutation to delete a like for a post
+      await context.db.PostLikes.delete({
+        account_id: likeAuthorAccountId,
+        post_id: postId,
+      });
+    } catch (e) {
+      console.log(`Failed to delete like from the database: ${e}`);
+    }
+  }
+
+  async function handleRepost( accountId, blockHeight, blockTimestamp, receiptId, repostContent) {
+
+    try {
+
+      console.log("Reposts are a WIP repostContent:", repostContent);
+
+      const content = JSON.parse(repostContent);
+
+      if (content[1]?.value?.type !== "repost") {
+        console.log("Skipping non-repost content", content);
+        return;
+      } 
+
+      console.log("Reposts are a WIP content:", content);
+      const postAuthor = content[1].key.path.split("/")[0];
+      const postBlockHeight = content[1].key.blockHeight;
+
+      console.log("Reposts are a WIP postAuthor:", postAuthor);
+      console.log("Reposts are a WIP blockHeight:", postBlockHeight);
+      
+
+      try {
+        const posts = await context.db.Posts.select(
+            { account_id: postAuthor, block_height: postBlockHeight },
+            1
+        );
+        console.log(`posts: ${JSON.stringify(posts)}`, posts);
+        if (posts.length == 0) {
+          return;
+        }
+
+        const post = posts[0];
+        const accountsReposted = post.accounts_reposted.length === 0 ? post.accounts_reposted : JSON.parse(post.accounts_reposted);
+        if (accountsReposted.indexOf(accountId) === -1) {
+          accountsReposted.push(accountId);
+        }
+
+       
+
+        try {
+          const repostData = {
+            post_id: post.id,
+            account_id: accountId,
+            content: JSON.stringify(content),
+            block_height: blockHeight,
+            block_timestamp: blockTimestamp,
+            receipt_id: receiptId,
+          };
+          // Call GraphQL mutation to insert a new repost
+          await context.db.Reposts.insert(repostData);
+          console.log(`Repost by ${accountId} has been added to the database`);
+
+          
+          // Call GraphQL mutation to update a post's reposted accounts list
+        await context.db.Posts.update(
```

Comment:
@jackson-harris-iii are we able to include the accountList in the `insert` operation above?

Reply:
this file is being removed from commit (stale implementation) no longer user ing accountList.